modules/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md

```diff
@@ -138,7 +138,8 @@ tags: {
 | :-- | :-- | :-- |
 | `location` | string | The location the resource was deployed into. |
 | `name` | string | The name of the user assigned identity. |
-| `principalId` | string | The principal ID of the user assigned identity. |
+| `clientId` | string | The client ID (application ID) of the user assigned identity. |
```

Comment:
@davidkarlsen have you regenerated the readme for the module? I can support you in this if you like?

Reply:
I hand-edited it, do you have a script to handle it?

Reply:
ah, found https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Generate%20module%20Readme

Reply:
hmm, still a bit blurry to me - I would appreciate help here.

Reply:
Hey @davidkarlsen, please use the following:

- Browse to the repo root path
   - `cd path/to/ResourceModules`
- Load the utility
   - `. ./utilities/tools/Set-ModuleReadMe.ps1`
- Update the readme using the loaded function
   - `Set-ModuleReadMe -TemplateFilePath 'modules/Microsoft.ManagedIdentity/userAssignedIdentities/deploy.bicep'`

Make sure you have the latest version of the Bicep CLI installed, for both Azure CLI and Azure PowerShell, ref [here](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install) for guidance